### ISSUE TITLE
Feature Request: Allow Override of Container Image Name for SurrealDB In Memory Container

### ISSUE THREAD

**Issue by jascenc1** [sebastianwessel/surrealdb-client-generator#60]
I’m using surql-gen (v2.6.0) in an environment where pulling images directly from Docker Hub is blocked due to a VPN/proxy. I have a SurrealDB image available in our Artifactory, but the tool tries to pull the latest image from Docker Hub.

### Feature Request:

It would be really useful to have an option to override the default SurrealDB container image name, like how Helm charts allow for container overrides. This would let users specify a custom image from an internal registry (e.g., Artifactory), making the tool more flexible in different network setups where Docker Hub isn’t accessible, and images need to come from an internal registry.

### Proposed Solution:

Add a config option in surql-gen to let users pull `surrealdb:latest` from an internal registry.

Example usage:

```bash
npx surql-gen -f schema.surql -o ./output --no-generate-client --container-registry myregistry.com/surrealdb/surrealdb:latest
```

That would update pass in the registry URL [here](https://github.com/sebastianwessel/surrealdb-client-generator/blob/main/src/database/db.ts#L19).

Thanks for considering this! The tool has been super helpful, and this feature would make it even better for different environments.

**Comment by hammo92** [comment]
Seems reasonable to me, should be a simple change. Any concerns @sebastianwessel?

**Comment by sebastianwessel** [comment]
no makes sense.
maybe, a short look into testcontainer config might be good - don´t know out of my mind if there are other helpfully options

**Comment by sebastianwessel** [comment]
Released version 2.6.1, which includes the config option. Thanks @hammo92

**Comment by jascenc1** [comment]
### Note for Private Registry Users:

If you're using a private registry, you’ll need to configure Testcontainers to pull the Ryuk image from your internal registry. You can do this by creating a `.testcontainers.properties` file in your user directory (windows: e.g., `C:/Users/YourUsername/.testcontainers.properties`) with the following configuration:

```bash
ryuk.container.image = yourregistry.com/ryuk:0.3.3
```
This ensures Testcontainers pulls the Ryuk image from your registry, which is necessary for container cleanup.

See [here](https://java.testcontainers.org/features/configuration/#disabling-ryuk:~:text=to%20workaround%20that.-,ryuk.container.image%20%3D%20testcontainers/ryuk%3A0.3.3,-Performs%20fail%2Dsafe) for more details
